sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -260,6 +213,94 @@ static const struct dmi_system_id sof_sdw_rt711_rt1308_rt715_quirk_table[] = {
 	{}
 };
 
+static const struct snd_soc_dapm_widget rt700_widgets[] = {
+	SND_SOC_DAPM_HP("Headphones", NULL),
+	SND_SOC_DAPM_MIC("AMIC", NULL),
+	SND_SOC_DAPM_SPK("Speaker", NULL),
+};
```

Comment:
I think it would be much better to first do the split and then add new codecs into new files directly.

Reply:
we'll squash all this anyways.